client/src/components/Learning/Content/Content.js

```diff
@@ -12,7 +12,10 @@ export default function Content(props) {
       setChapterIndex(chapters);
       e.setAttribute("id", e.innerText.split(" ").join(""));
     });
-  }, [setChapterIndex]);
+
+    if (headings.length < 1) setChapterIndex([]);
+    console.log("first");
```

Comment:
Please remove this console.

Reply:
Oh sorry, I forgot to remove it! 😅 Will fix it in next commit

Reply:
Sure.